starter-code/src/components/FoodList.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import foods from '../foods.json'
+import FoodBox from './FoodBox'
+import AddFood from './AddFood'
+import SearchBar from './SearchBar'
+import ListElement from "./TodayFoods"
+
+
+class FoodList extends Component {
+  constructor(){
+    super() 
+    this.state = {
+     foods:foods,
+     appear: false,
+     todaysFoods:[]
+     
+
+    }
+  }
+  filteredListHandler = (filteredList) =>{
+    console.log(filteredList)
```

Comment:
¿Qué estáis haciendo en esta función?

Si lo leemos despacio estáis guardando en una variable llamada `foodCopy` una copia del estado foods. 
Luego a esta variable la resignáis el valor del argumento por la que estáis llamando a la función, por tanto estáis haciendo una copia para nada.
Y luego estáis actulizando estado foods por foodCopy que a su vez es el mismo valor que tiene el argumento de la función. 
Entonces podríamos dejar está función algo así:
`this.setState({foods:filteredList})`